VAMobile/src/api/facilities/getFacilitiesInfo.tsx

```diff
@@ -24,5 +24,6 @@ export const useFacilitiesInfo = () => {
     meta: {
       errorName: 'getFacilitiesInfo: Service error',
     },
+    staleTime: Infinity,
```

Comment:
This was being called every time the Health screen was opened. Don't think this is likely to change in a user's session, so I set it to never be stale